genome_annotation_tools_3.yml

```diff
@@ -94,7 +97,7 @@ tools:
   tool_panel_section_label: "Assembly"
 
 - name: cdhit
-  owner: jjohnson
+  owner: bebatut
```

Comment:
I think this should be IUC https://github.com/galaxyproject/tools-iuc/pulls?q=is%3Apr+cdhit+is%3Aclosed

Reply:
ah yes sorry, I missed this one, I will correct this